src/EnergyPlus/DataZoneEquipment.hh

```diff
@@ -477,15 +481,28 @@ namespace DataZoneEquipment {
         void adjustLoads(EnergyPlusData &state, int zoneNum, int equipTypeNum);
     };
 
-    struct ZoneEquipmentMixer : ZoneEquipmentSplitterMixer
+    struct ZoneMixer : ZoneEquipmentSplitterMixer
```

Comment:
Add a mid-level class so the two mixers can share the `setOutletConditions` function.